dockers/docker-dhcp-server/cli-plugin-tests/test_show_dhcp_server.py

```diff
@@ -55,3 +55,57 @@ def test_show_dhcp_server_ipv4_lease_client_not_in_fdb(self, mock_db):
         result = runner.invoke(show_dhcp_server.dhcp_server.commands["ipv4"].commands["lease"], ["Vlan1001"], obj=db)
         assert result.exit_code == 0, "exit code: {}, Exception: {}, Traceback: {}".format(result.exit_code, result.exception, result.exc_info)
         assert result.stdout == expected_stdout
+
+    def test_show_dhcp_server_ipv4_range_without_name(self, mock_db):
+        expected_stdout = """\
+Range    IP Start    IP End     IP Count
+-------  ----------  ---------  ----------------------
+range1   100.1.1.3   100.1.1.5  3
+range2   100.1.1.9   100.1.1.8  range value is illegal
+"""
+        runner = CliRunner()
+        db = clicommon.Db()
+        db.db = mock_db
+        result = runner.invoke(show_dhcp_server.dhcp_server.commands["ipv4"].commands["range"], [], obj=db)
+        assert result.exit_code == 0, "exit code: {}, Exception: {}, Traceback: {}".format(result.exit_code, result.exception, result.exc_info)
+        assert result.stdout == expected_stdout
+
+    def test_show_dhcp_server_ipv4_range_with_name(self, mock_db):
+        expected_stdout = """\
+Range    IP Start    IP End       IP Count
+-------  ----------  ---------  ----------
+range1   100.1.1.3   100.1.1.5           3
+"""
+        runner = CliRunner()
+        db = clicommon.Db()
+        db.db = mock_db
+        result = runner.invoke(show_dhcp_server.dhcp_server.commands["ipv4"].commands["range"], ["range1"], obj=db)
+        assert result.exit_code == 0, "exit code: {}, Exception: {}, Traceback: {}".format(result.exit_code, result.exception, result.exc_info)
+        assert result.stdout == expected_stdout
+
+    def test_show_dhcp_server_ipv4_range_wrong_data(self, mock_db):
+        expected_stdout = """\
+Range    IP Start    IP End     IP Count
+-------  ----------  ---------  ----------------------
+range2   100.1.1.9   100.1.1.8  range value is illegal
+"""
+        runner = CliRunner()
+        db = clicommon.Db()
+        db.db = mock_db
+        result = runner.invoke(show_dhcp_server.dhcp_server.commands["ipv4"].commands["range"], ["range2"], obj=db)
+        assert result.exit_code == 0, "exit code: {}, Exception: {}, Traceback: {}".format(result.exit_code, result.exception, result.exc_info)
+        assert result.stdout == expected_stdout
+
+    def test_show_dhcp_server_ipv4_range_single_ip(self, mock_db):
+        expected_stdout = """\
+Range    IP Start    IP End       IP Count
+-------  ----------  ---------  ----------
+range1   100.1.1.3   100.1.1.5           3
```

Comment:
why expected output is range1 and not single, but in line 108 invoke is checking range3?